test/pod_performance/manifest.diff

```diff
@@ -0,0 +1,16 @@
+11083,11089c11083,11089
+<     # - "pod"
+<     # podOptions:
+<     #   namespaceSelector:
+<     #     matchExpressions:
+<     #       - key: kubernetes.io/metadata.name
+<     #         operator: NotIn
+<     #         values: [ kube-system, kueue-system ]
+---
+>       - "pod"
+>       podOptions:
+>         namespaceSelector:
+>           matchExpressions:
+>             - key: kubernetes.io/metadata.name
+>               operator: NotIn
+>               values: [ kube-system, kueue-system ]
```

Comment:
we just need to uncomment the first line. By default, the podOptions include the matchExpressions, if the user doesn't explicitly defines them.